pages/advanced/content/fr/index.js

```diff
@@ -10,4 +10,6 @@ module.exports = {
     require('./size'),
     require('./ssr'),
   ].filter(mod => Boolean(mod.title)),
+  contributorsTitle: 'Contributeurs:',
```

Comment:
I'd prefer to not have this in every file. Could we find another place to do this? I have an idea if you want me to pick this up from here.